### DISCUSSION TITLE
Can we upload custom bms without dragging a folder?

### DISCUSSION THREAD

**Discussion by kyu28** [bemusic/bemuse#674] [Q&A] [answered]
Sometimes I play bemuse on my android phone which is able to upload file to the browser, but bemuse only supports dragging folders instead of direct uploading file. Because of this I can't play custom bms on my phone. Is there a solution to upload custom bms without dragging a folder?

**Comment by dtinth** [comment, marked as answer]
~~Tracking in https://github.com/bemusic/bemuse/issues/711~~ Now possible in v49.2.0!
